src/targets/gpu/rocblas.cpp

```diff
@@ -41,6 +47,34 @@ rocblas_handle_ptr create_rocblas_handle_ptr(hipStream_t s)
     return rb;
 }
 
+const std::unordered_set<std::string>& get_rocblas_fp32_archs()
+{
+    static std::unordered_set<std::string> supported_archs{"gfx908", "gfx90a"};
+    return supported_archs;
+}
+
+bool get_compute_fp32_flag()
+{
+    bool compute_fp32 = false;
+#if ROCBLAS_VERSION_MAJOR >= 2 && ROCBLAS_VERSION_MINOR >= 38
+    const auto device_name =
+        migraphx::trim(migraphx::split_string(migraphx::gpu::get_device_name(), ':').front());
```

Comment:
No need to use `migraphx::` namespace on these functions.

Reply:
Done